Reference body not data in the Python requests.post call

wjohnson opened the pull request:
They data variable is incorrectly referenced in the requests.post call.  It should be body since it encodes the data variable into json.

azurecla commented:
Hi __@henrybeen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@henrybeen : Thanks for your contribution to the Azure documentation! The author, @dhanyahk, has been notified to review your proposed change.

dhanyahk commented:
Looks good.

wjohnson committed:
Reference body not data in the requests.post call

They data variable is incorrectly referenced in the requests.post call.  It should be body since it encodes the data variable into json.
